redhat-cop/uncontained.io
# Fix pipeline

#### What is this PR About?

Fixed up the pipeline to get production deployments rolling again.

#### How should we test or review this PR?

1. Run applier. Pipeline should succeed through stage and then fail at production promote due to missing secret. (it works with the secret, I swear)
2. Build local dev image:
```
docker build -f container-images/local-dev/Dockerfile -t uncontained-local-dev .
```
3. Run local build tests
```
docker run -u $(id -u) -v ${PWD}:/uncontained.io:Z -it --net=host uncontained-local-dev run build
docker run -u $(id -u) -v ${PWD}:/uncontained.io:Z -it --net=host uncontained-local-dev test
```

#### Is there a relevant Trello card or Github issue open for this?

Addresses #132 and #133 

#### Who would you like to review this?

cc: @redhat-cop/cant-contain-this

'''''

https://github.com/redhat-cop/uncontained.io/blob/master/CONTRIBUTING.md[contributing
guidelines]?
the uncontained.io guides?

'''''

*Please note: we may close your PR without comment if you do not check
the boxes above and provide ALL requested information.*
